src/site/content/en/handbook/inclusion-and-accessibility/index.md

```diff
@@ -47,23 +46,23 @@ While _written language_ and _character_ are slightly less precise, they should
 If you must use a word that your audience is unlikely to know, provide a definition using the [Key-term Aside component](/handbook/web-dev-components#asides).
 
 ## Make information easy to find
-Headings [shouldn't skip levels](/heading-levels). Also, don't add custom styles to headings—it makes the page hierarchy more difficult to follow for sighted readers.
+[Don't skip heading levels](/heading-levels). Don't add custom styles to headings—it makes the page hierarchy more difficult to follow.
```

Comment:
removed "sighted readers" since this is difficult for all (sighted, those using screen readers, crawlers, etc)

Reply:
SGTM

Reply:
Fixed in Google Doc